### ISSUE TITLE
[CWI] Add "Found in build" link during Release

### ISSUE THREAD

**Issue by mbhoek** [MicrosoftPremier/VstsExtensions#146]
As part of our CI/CD, we deploy a build for manual testing using a (classic) release stage.
In this stage we use "Create Work Item" to automatically create a work item to capture test results.

It would be helpful if we could add the "Found in build" artifact link for the build used in this release, e.g.
![image](https://user-images.githubusercontent.com/420399/120895916-65415b80-c61f-11eb-8991-5024b3ea510e.png)

At the moment we have "Associate Work Item With Current Build or Release" enabled, but it looks like this only adds a "Integrated in release stage" link if used in a release stage (only viewable on the Links tab of the work item).

A solution could be to split this option into two checkboxes: "Associate Work Item with Build" and "Associate Work Item with Release", which can be individually enabled.

Thanks for your time!

**Comment by ReneSchumacher** [comment]
Hi @mbhoek,

thanks for the feedback. I'll have a bit time to spare over the next two weeks to look at this and see what I can do. I'm currently not sure what exact link we use when linking the new work item with the current build or release. Let me check, and I will get back to you with a proposed solution.

I'd rather not have two different check boxes for build and release, because it doesn't make sense, e.g., to check "Associate with Release" if the task runs as part of a build. However, it might be useful to have options for different link types.

**Comment by ReneSchumacher** [comment]
Hi again,

the new version is finally being rolled out. It should be published to the marketplace within the hour. You can now specify the link type (Build, Found in build, or Integrated in build) for builds. The option is ignored for classic release because there is only one artifact link type available (Integrated in release environment).

**Comment by mbhoek** [comment]
Thanks @ReneSchumacher,

Unfortunate that the options are ignored in classic release (our use case) but I guess that's because of limitations of the classic release pipelines which are outside of your control. Guess we have another reason to push for the new pipelines.

Cheers!

**Comment by ReneSchumacher** [comment]
Ah, I see. We cannot do much about that. Releases only allow one link type. Are you trying to link the build(s) that is/are deployed with the release? Currently, we only link the work item to the release environment. It's a bit more work, but we could read the builds that go into the release and link the work item to those.

**Comment by mbhoek** [comment]
Yes, that's exactly what we're trying to do; link to the builds that go into the release. Apologies if my original post did not make that clear; the screenshot I included is a representation of how we currently do it manually. What I am trying to achieve is create that exact link using CWI in a (classic) release deployment.
